compose.py

```diff
@@ -0,0 +1,297 @@
+COPY --chown=1000:1000 --from=full /opt/tritonserver/nvidia_entrypoint.sh .
+ENTRYPOINT ["/opt/tritonserver/nvidia_entrypoint.sh"]
+'''
+    with open(os.path.join(ddir, dockerfile_name), "a") as dfile:
+        dfile.write(df)
+
+
+### Create container with docker build
+def build_docker_container(ddir, dockerfile_name, container_name):
+    p = subprocess.Popen(['docker', 'build', '-t', container_name, '-f', \
+        os.path.join(ddir, dockerfile_name), '.'])
+    p.wait()
+    fail_if(p.returncode != 0, 'docker build {} failed'.format(container_name))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    group_qv = parser.add_mutually_exclusive_group()
+    group_qv.add_argument('-q',
+                          '--quiet',
+                          action="store_true",
+                          required=False,
+                          help='Disable console output.')
+    group_qv.add_argument('-v',
+                          '--verbose',
+                          action="store_true",
+                          required=False,
+                          help='Enable verbose output.')
+    parser.add_argument('--build-name',
+                        type=str,
+                        required=False,
+                        help='Build name. Default is "tritonserver".')
+    parser.add_argument(
+        '--work-dir',
+        type=str,
+        required=False,
+        help=
+        'Generated dockerfiles are placed here. Default to current directory.'
+    )
+    parser.add_argument(
+        '--version',
+        type=str,
+        required=False,
+        help=
+        'The Triton version. If not specified defaults to the value in the TRITON_VERSION file.'
+    )
+    parser.add_argument(
+        '--container-version',
+        type=str,
+        required=False,
+        help=
+        'The Triton container version to build. If not specified the container version will be chosen automatically based on --version value.'
+    )
+    parser.add_argument(
+        '--upstream-container-version',
+        type=str,
+        required=False,
+        help=
+        'The upstream container version to use for the build. If not specified the upstream container version will be chosen automatically based on --version value.'
+    )
+    parser.add_argument(
+        '--enable-gpu',
+        action="store_true",
+        required=True,
+        help=
+        'Flag to enable gpus for the build. Must be specified since only-cpu is not supported'
+    )
+    parser.add_argument(
+        '--backend',
+        action='append',
+        required=False,
+        help=
+        'Include specified backend in build as <backend-name>. To have multiple backends built, specify --backend <foo> --backend <bar>'
+    )
+    parser.add_argument(
+        '--repoagent',
+        action='append',
+        required=False,
+        help='Include specified repo agent in build as <repoagent-name>')
+    FLAGS = parser.parse_args()
+    
```

Comment:
Make sure FLAGS.enable_gpu is true.

Reply:
I specified `required=True` so It will always raise an error here.